app/javascripts/graphiql.jsx

```diff
@@ -6,6 +6,10 @@ import {AppConfig} from './config';
 import './css/graphiql.css'
 import 'graphiql/graphiql.css'
 
-const config = new AppConfig("graphiql")
-
-ReactDOM.render(<GraphiQLTool config={config} />, document.getElementById('graphiql-tool'));
\ No newline at end of file
+module.exports = {
+  bootstrap: function (bootstrapOptions = {}) {
+    this.bootstrapOptions = bootstrapOptions;
+    const config = new AppConfig("graphiql", bootstrapOptions);
+    ReactDOM.render(<GraphiQLTool config={config} />, document.getElementById('graphiql-tool'));
```

Comment:
What do you think about parametrizing `document.getElementById('graphiql-tool')` as well? If it would be used standalone, i could imagine that people would also want to place in a different parent.